configuration/jsr223-jython.md

```diff
@@ -0,0 +1,117 @@
+
+### Mac and Windows or Manual Installation
+
+Set the `EXTRA_JAVA_OPTS` environment variable to the value described in the Linux configuration. Note that on Windows the environment variable value should be set as a single line (without backslashes to combine the line fragments).
+
+## Configuration Testing
+
+In the `automation/jsr223` directory create an empty file called `test.py`.
+
+Watch the openHAB log file carefully ([Karaf logging]({{base}}/administration/logging.html) or viewing the `openhab.log` file directly). You should see a log line with information similar to:
+
+	... [INFO ] [s.i.GenericScriptEngineFactory:28   ] - Activated scripting support for ECMAScript
+	... [INFO ] [s.i.GenericScriptEngineFactory:28   ] - Activated scripting support for python
+		...
+	... [INFO ] [.a.m.s.r.i.l.ScriptFileWatcher:150  ] - Loading script 'test.py'
+
+> NOTE: ECMAScript is Javascript
+
+Look for any potentially related error messages.
+
+To enable debug logging, use the [Karaf logging]({{base}}/administration/logging.html) commands to enabled debug logging for the automation functionality:
+
+```
+log:set DEBUG org.eclipse.smarthome.automation
+```
+
+## Script Examples
+
+Jython scripts provide access to almost all the functionality in an openHAB runtime environment. As a simple example, the following script logs "Hello, World!" into the openhab log file. Note that `print` will usually not work since the output has no terminal to display the text. One exception is when the openHAB server is running from the Eclipse IDE.
+
+```python
+from org.slf4j import LoggerFactory
+
+LoggerFactory.getLogger("org.eclipse.smarthome.automation.examples").info("Hello world!")
+```
+
+The openHAB server uses the [SLFJ](https://www.slf4j.org/) library for logging. Jython can [import Java classes](http://www.jython.org/jythonb Depending on the openHAB logging configuration, you may need to prefix logger names with `org.eclipse.smarthome.automation` for them to show up in the log file (or you modify the logging configuration).
+
+> NOTE: Be careful with using wildcards when importing Java packages (e.g., `import org.sl4j.*`). This will work in some cases, but it might not work in some situations. It's best to use explicit imports with Java packages. For more details, see the Jython documentation on [Java package scanning](http://www.jython.org/jythonbook/en/1.0/ModulesPackages.html#java-package-scanning).
+
+The script then uses the [LoggerFactory](https://www.slf4j.org/apidocs/org/slf4j/Logger.html) to obtain a named logger and then logs a message like:
+
+```
+	... [INFO ] [.smarthome.automation.examples:-2   ] - Hello world!
+```
+
+Notice that no rules were required for this simple script. This can be a useful way to experiment with simple Jython code before defining rules or other mroe advanced functionality.
```

Comment:
mroe -> more

Rather, that sentence would read better as: "This can be a useful way to experiment with simple Jython code before defining advanced rules and functionality."